.github/workflows/dapr-perf.yml

```diff
@@ -23,9 +23,9 @@
 name: dapr-perf
 
 on:
-  # Run every 2 hours
+  # Run every 6 hours
   schedule:
-    - cron: "0 */2 * * *"
+    - cron: "0 */6 * * *"
```

Comment:
Only My personal opinion: Usually, performance tests are exhaustive, as they grow up, and they tend to take many resources to run. I would, in general, prefer having Perf suite NOT run more than once in 24 hours.

Reply:
Sure. I think we may be creating a new CI for this which will probably run once a week. This can be changed as per the need. But, I do get the idea of not running the perf test too frequently, as it will not serve too many benefits. For now I am making it 12 hours.